.buildkite/pipeline.yml

```diff
@@ -168,3 +168,28 @@ steps:
     plugins:
       - junit-annotate#v2.4.1:
           artifacts: build/TEST-go-integration*.xml
+
+  - group: "K8s tests"
+    key: "k8s-tests"
+    steps:
+      - label: "K8s tests: {{matrix.k8s_version}}"
+        env:
+          K8S_VERSION: "v{{matrix.k8s_version}}"
+          KIND_VERSION: "v0.20.0"
+        command: ".buildkite/scripts/steps/k8s-tests.sh"
+        artifact_paths:
+          - "build/TEST-**"
+          - "build/diagnostics/*"
+          - "coverage.out"
+        agents:
+          provider: "gcp"
+          image: "family/core-ubuntu-2204"
```

Comment:
Does this equal to `ubuntu-22 && immutable` that we specify in Jenkinsfile?

Reply:
In general, these are both Ubuntu 22.04 images. The main difference is in preinstalled agents and tools because we migrate to buildkite

Reply:
I think these are more or less equal ubuntu 22.04 VMs, except for the differences in preinstalled tools because buildkite requires another tool set.